src/components/TodoItem.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { KEYCODE } from '../utils/const';
+
+export class TodoItem extends React.PureComponent {
+  state = {
+    inEditMode: false,
```

Comment:
```suggestion
    isEditMode: false,
```

Reply:
this kind of variable is usually named like this

Reply:
ok, renamed.